ci-operator/step-registry/insights-qe/iqe-cost-management-plugin/insights-qe-iqe-cost-management-plugin-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: insights-qe-iqe-cost-management-plugin
+  commands: insights-qe-iqe-cost-management-plugin-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  from: cli
+  credentials:
+  - namespace: test-credentials
+    name: my-secret
```

Comment:
Where does this secret live in vault?

Who has access to this location?

I think we need to make sure that multiple people on our team have access to this secret location, I think the [insights-qe secret collection](https://vault.ci.openshift.org/ui/vault/secrets/kv/list/selfservice/insights-qe/) would be a good place to hold this ci pull secret

Reply:
Moved the credentials to the insights-qe collection.

Reply:
@ascerra please check the insights-qe collection, looks like there is no permission to get to the secrets on this collection. 

 * could not run steps: step cost-management-interop-aws failed: failed to create credentials: could not read source credential: secrets "my-secret" is forbidden: User "system:serviceaccount:ci:ci-operator" cannot get resource "secrets" in API group "" in the namespace "test-credential" 
[36mINFO[0m[2023-06-21T16:51:56Z] Reporting job state 'failed' with reason 'executing_graph:step_failed:utilizing_lease:executing_test:executing_multi_stage_test'

Reply:
@ascerra switched to insights-qe collection for vault credentials.